docker_registry_client/DockerRegistryClient.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 
-from docker_registry_client._BaseClient import BaseClient
```

Comment:
I recently took over the maintenance of this project for my company and I'm a bit rusty on my python knowledge.  What are the ramifications of these import statements?  Will they break backward compatibility with older python versions?